test_accounts/appeals_status_test_accounts.md

```diff
@@ -0,0 +1,27 @@
+# Sandbox Test Accounts for Appeals Status API (v1)
+
+This document provides test account information for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current). The API returns synthetic VA records for these test accounts in the sandbox environment.
+
+## Authentication and Authorization
+
+The authentication model for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current) uses OAuth 2.0/OpenID Connect. The following authorization models are supported:
+- [Authorization code flow](https://developer.va.gov/explore/api/appeals-status/authorization-code)
+- [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appeals-status/client-credentials)
+
+Login information for all test accounts can be found on the [developer portal](link-will-go-here-once-page-created).
+
+## Veteran Test Accounts
+
+These accounts can be used to test various data and response scenarios for the `GET /appeals` endpoint. 
+
+An `icn` parameter and an `X-VA-User` header are required when making a request. The `icn` should match the value in the ICN column, and the `X-VA-User` should match the value in the ID.me Username or Login.gov Username column (depending on the authentication method used).
+
+| Sandbox Response                                                    | ICN               | ID.me Username                 | Login.gov Username        | First Name | Last Name |
+|:--------------------------------------------------------------------|:------------------|:-------------------------------|:--------------------------|:-----------|:----------|
+| Higher Level Review (1), Supplemental Claim (1), Legacy Appeals (2) | 1012832025V743496 | va.api.user+idme.025@gmail.com | va.api.user+025@gmail.com | Wesley     | Ford      |
```

Comment:
I just realized that you may have meant A-Z order for the accounts themselves rather than the order of the columns, which is what I was commenting on above. Sorry if I misunderstood!

My hesitation to order the accounts A-Z based on username is that I believe a more useful order for consumers coming to this page and looking for test data for their testing scenarios will be _most data returned_ → _least data returned_ → _error responses_. As you'll see in the current table, that new ordering paradigm conflicts with the older paradigm of A-Z ordering by username.